src/Cake.Hosts/HostsFile.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Cake.Hosts
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class HostsFile
```

Comment:
I'd rename that to be `HostsRecord` as this really represents a line in hosts file, not the entire hosts file.